doc/source/building/working_with_images/vmx_setup_for_ec2.rst

```diff
@@ -0,0 +1,129 @@
+.. _setup_for_ec2:
+
+KIWI image description for Amazon EC2
+=====================================
+
+.. sidebar:: Abstract
+
+   This page provides further information for handling
+   vmx images built with KIWI and references the following
+   articles:
+
+   * :ref:`vmx`
+
+A virtual disk image which is able to boot in the Amazon EC2
+cloud framework has to comply the following constraints:
+
+* Xen tools and libraries must be installed
+* cloud-init must be installed
+* cloud-init configuration for amazon must be provided
+* grub bootloader modules for Xen must be installed
+* AWS tools must be installed
+* Disk size must be set to 10G
+* Kernel parameters must allow for xen console
+
+In order to meet this requirements add or update the KIWI image
+description as follows:
+
+1. Software packages
+
+   Make sure to add the following packages to the package list
+
+   .. note::
+ 
+      Package names used in the following list matches the
+      package names of the SUSE distribution and might be different
+      on other distributions.
+
+   .. code:: xml
+
+      <package name="aws-cli"/>
+      <package name="grub2-x86_64-xen"/>
+      <package name="xen-libs"/>
+      <package name="xen-tools-domU"/>
+      <package name="cloud-init"/>
+
+2. Image Type definition
+
+   Update the vmx image type setup as follows
+
+   .. code:: xml
+
+      <type image="vmx"
+            filesystem="ext4"
+            initrd_system="dracut"
+            bootloader="grub2"
+            kernelcmdline="console=xvc0 multipath=off net.ifnames=0"
+            boottimeout="1"
+            devicepersistency="by-label"
+            firmware="ec2">
+        <size unit="M">10240</size>
+      </type>
+
+3. Cloud Init setup
+
+   Cloud init is a service which runs at boot time and allows
+   to customize the system by activating one ore more cloud init
+   modules. For Amazon EC2 the following :file:`etc/cloud/cloud.cfg`
```

Comment:
* Is there a "/" before "etc/cloud" missing?
* I would add "... the following **configuration file** ..."